pkg/cmd/server/origin/master_config.go

```diff
@@ -293,7 +293,7 @@ var (
 		"ProjectRequestLimit",
 		"OriginNamespaceLifecycle",
 		"PodNodeConstraints",
-		"JenkinsBootstrapper",
+		//"JenkinsBootstrapper",
```

Comment:
when we're ready to turn it on, it will be part of a new release that enables the feature.  we're not going to ship doc to customers about how to turn on a feature we deemed not ready for consumption.